synapse/rest/admin/users.py

```diff
@@ -275,6 +276,21 @@ async def on_PUT(self, request, user_id):
                     await self.auth_handler.add_threepid(
                         user_id, threepid["medium"], threepid["address"], current_time
                     )
+                    if (
+                        self.hs.config.email_enable_notifs
+                        and self.hs.config.email_notif_for_new_users
+                    ):
+                        await self.pusher_pool.add_pusher(
+                            user_id=user_id,
+                            access_token=None,
```

Comment:
Can this access token be `None`? I'm skeptical this will work as expected due to the care that is taken with it in the standard API:

https://github.com/matrix-org/synapse/blob/v1.12.0/synapse/handlers/register.py#L633-L638

Reply:
The admin user has no token of the affected user. What will be the alternativ? Is it not possible for admin users to create the pusher? The access_token is needed for removing the pusher.

https://github.com/matrix-org/synapse/blob/v1.12.0/synapse/push/pusherpool.py#L170-L172

Reply:
I'm not sure what the ramifications of doing this would be. I'm going to set the review flag on this and see if others have thoughts.

Reply:
The database table is fine with it being null, and I *think* the only thing we use it for is to delete the pusher when the associated device is logged out. I'd be slightly surprised if that also happened to email push, but might explain why some people are reporting email push as flakey?